client/lib/invites/actions.js

```diff
@@ -51,7 +51,9 @@ export function fetchInvite( siteId, inviteKey ) {
 		} );
 
 		if ( error ) {
-			analytics.tracks.recordEvent( 'calypso_invite_validation_failure' );
+			analytics.tracks.recordEvent( 'calypso_invite_validation_failure', {
+				error: error.error
```

Comment:
Should we see if `error.error` exists before using it, or are we assuming that it will always be safe?